src/G2/Equiv/Uninterpreted.hs

```diff
@@ -19,7 +19,6 @@ addFreeVarsAsSymbolic eenv = let xs = freeVars eenv eenv
 
 -- | changing the signature of addFreeTypes so that we have 
 -- we want to modify the rewrite-rules passed in checkrule 
```

Comment:
Not sure what this is saying- might have been a note to yourself?

Reply:
I believe so.